### ISSUE TITLE
PEM profile property connectionOptimizationService value: "none' is invalid

### ISSUE THREAD

**Issue by azahajkiewicz** [f5devcentral/f5-as3-config-converter#36]
### Environment
 * Application Services Version: 1.11.0
 * BIG-IP Version: 15.x

### Summary
PEM property 

### Steps To Reproduce
1. Convert following config:
```conf
ltm virtual /tenant_1/application_1/test_vs {
    description "Created by Web Configuration Utility for PEM Listener test_vs"
    destination /tenant_1/application_1/0.0.0.0:80
    ip-protocol udp
    mask any
    profiles {
        /Common/classification_pem {
            context clientside
        }
        /tenant_1/application_1/test_pem_profile {
            context clientside
        }
        /Common/udp_decrement_ttl { }
    }
    source 10.10.10.100/32
    source-address-translation {
        pool /Common/CGNAT_pool
        type lsn
    }
    translate-address disabled
    translate-port enabled
    vlans-enabled
}
ltm virtual-address /tenant_1/application_1/0.0.0.0 {
    address any
    arp disabled
    icmp-echo disabled
    mask any
    traffic-group /Common/traffic-group-1
}
pem profile radius-aaa /tenant_1/application_1/test_radius_profile {
    app-service none
    defaults-from /Common/radiusaaa
    description "Created by Web Configuration Utility for PEM Control Virtual /Common/radiusacct"
    password calypso
    shared-secret xxx
}
pem profile spm /tenant_1/application_1/test_pem_profile {
    app-service none
    description "Created by Web Configuration Utility for PEM Listener ..."
    fast-pem disabled
    fast-vs-name none
    global-policies-high-precedence {
        /Common/Parental_policy
    }
}
```

2. Observe the following error message:

Declaration created by Charon:
```
            "test_pem_profile": {
                "remark": "xxx",
                "connectionOptimizationEnabled": false,
                "connectionOptimizationService": {
                    "use": "none"                                        # invalid reference          
                },
                "policiesGlobalHighPrecedence": [
                    {
                        "bigip": "/Common/Shared/Parental_policy"
                    }
                ],
                "class": "Enforcement_Profile"
            },
```

Error returned: 

```out
{
    "code": 422,
    "errors": [
        "/tenant_1/application_1/test_pem_profile/connectionOptimizationService/use: contains path to non-existent object none"
    ],
    "declarationFullId": "",
    "message": "declaration is invalid"
}
```

### Expected Behavior
 Not sure, but  "connectionOptimizationService": { "use": "none"  }   is invalid.

**Comment by mdditt2000** [comment]
Created Jira CHARON-320 for PM tracking

**Comment by mdditt2000** [comment]
Merge branch '320-pem-property' into 'develop' - resolved in ACC 1.13, Closing
